### ISSUE TITLE
Feature: Add Pizza CLI to the Homebrew Core Formulae Repository

### ISSUE THREAD

**Issue by udaypatel1** [open-sauced/pizza-cli#225]
### Suggested solution

In it's current state and to my knowledge, pizza-cli is not natively supported / merged as a [core formulae](https://github.com/Homebrew/homebrew-core) for homebrew.

It'd be nice to just `brew install pizza-cli` or `brew install pizza`! Might allow for higher adoption and usage rates.

You can find more information on how to contribute and get a project accepted [here](https://github.com/Homebrew/homebrew-core/blob/555c53a09cbffc3f1f198af032402085f770587f/CONTRIBUTING.md)

We'd also likely need to, in the future, set up CI to open a PR against the homebrew core library when pizza-cli has new version releases.

**Comment by jpmcb** [comment]
Yes, in the future, that would be great. We can possibly look at this when we have more engineering resources on hand!

**Comment by udaypatel1** [comment]
...does OpenSauced have any plans of increasing said engineering resources on hand? 👀
